### PR TITLE
fix: add foxy4096

### PR DESCRIPTION
**Pull request by foxy4096**

### CODE REVIEW

File: public/data/foxy4096.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/Foxy4096",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (krishguptadev) [inline public/data/foxy4096.json:18]:
```suggestion

```

File: public/data/foxy4096.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "url": "https://github.com/foxy4096",
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/Foxy4096",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/foxy4096.json:17]:
```suggestion
}
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: public/data/foxy4096.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/Foxy4096",
+        "icon": "twitter"
+      }
+    ]
+}
+
```
Comment (krishguptadev) [inline public/data/foxy4096.json:18]: Not yet fixed.

```suggestion

```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: public/data/foxy4096.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/foxy4096",
+        "icon": "github"
+      },
+      {
+        "name": "Twitter",
+        "url": "https://twitter.com/Foxy4096",
+        "icon": "twitter"
+      }
+    ]
+}
```
Comment (krishguptadev) [inline public/data/foxy4096.json:17]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see the comments above.

**Review by schmelto — APPROVED** [review]
LGTM

image

**Review by krishguptadev — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
LGTM!

**Review by Panquesito7 — COMMENTED** [review]
LGTM! 🚀

**Pull request merged**